### PR TITLE
Do not silently swallow Exception ...

### PR DESCRIPTION
**Pull request by butonic**
Should help debugging. There are probably other places where we could log this ... 

cc @MorrisJobke @tariqsiddiqui @icewind1991

### CODE REVIEW

**Comment by icewind1991** [comment]
:+1:

File: apps/files/ajax/delete.php [hunk 1]
```diff
@@ -61,6 +61,7 @@
 			$success = false;
 		}
 	} catch (\Exception $e) {
+		\OC::$server->getLogger()->error($e->getMessage(), ['app'=>'files']);
```
Comment (MorrisJobke) [inline apps/files/ajax/delete.php]: There are sometimes exceptions with a quite verbose name but without message. please use `logException()`of Logger (was added in 8.2.0)

**Comment by DeepDiver1975** [comment]
Pointless to bring this into master - this file will die

**Comment by PVince81** [comment]
File will die in 3...2...1... well, as soon as https://github.com/owncloud/core/pull/16902 is merged

**Pull request closed without merging**